pull_request_template.md

```diff
@@ -1,6 +1,23 @@
 # Purpose of this PR
 
+
+## Type of change
+
+- [ ] Bug fix 
+- [ ] Refactoring
+- [ ] New feature 
+- [ ] Minor change
```

Comment:
for me, it isn't entirely clear what a "minor change" or "fundamental change" is, in particular as "bug fix" of "refactoring" are alternative options.

Reply:
for me a "minor change" comes with less danger for the whole system. A "fundamental change" results in different results/model behaviour, etc.

Reply:
then maybe add a short explanation or write it explicitly "minor change: does not alter model behavior". Because a bug fix could also result in different model results, but as I understand you now, the idea is that you can and should tick multiple boxes here. And of course, if it changes model behavior, one should state what it changes :)